packages/SwingSet/src/vats/timer/timeMath.js

```diff
@@ -6,7 +6,6 @@ import { Nat } from '@agoric/nat';
 import { fit } from '@agoric/store';
 
 import { RelativeTimeShape, TimestampShape } from './typeGuards.js';
-import './types.js';
```

Comment:
I'm confused, how can files that transitively import `timeMath.js` now get the ambient (now global) types defined in `types.d.ts` ?

Reply:
Within this package the `jsconfig.json` is picking up the global types in `types.d.ts`.

Outside this package they come along with `exported.js`. That's why https://github.com/Agoric/agoric-sdk/pull/5855#discussion_r933590277

Reply:
> I am still concerned that other packages which don't have a *.ts in their include will simply have these type silently default to any, but I don't have time to validate this one way or another.

Does this assuage your concern?
<img width="738" alt="Screen Shot 2022-07-29 at 1 29 37 PM" src="https://user-images.githubusercontent.com/21505/181838626-3310a159-6fde-429a-acc6-36ae09582142.png">

> I believe a triple-slash directive would have been safer in this case.

I tried that and couldn't get it to work. Maybe could be done with more time but I struck upon this solution and I don't yet understand what's deficient in it.